packages/kbn-securitysolution-list-constants/index.ts

```diff
@@ -70,30 +72,76 @@ export const MAXIMUM_SMALL_VALUE_LIST_SIZE = 65536;
 
 export const MAXIMUM_SMALL_IP_RANGE_VALUE_LIST_DASH_SIZE = 200;
 
-/** ID of trusted apps agnostic list */
-export const ENDPOINT_TRUSTED_APPS_LIST_ID = 'endpoint_trusted_apps';
+/**
+ * List definitions for Endpoint Artifact
+ */
+export const ENDPOINT_ARTIFACT_LISTS = deepFreeze({
+  trustedApps: {
+    id: 'endpoint_trusted_apps',
+    name: 'Endpoint Security Trusted Apps List',
+    description: 'Endpoint Security Trusted Apps List',
+  },
+  eventFilters: {
+    id: 'endpoint_event_filters',
+    name: 'Endpoint Security Event Filters List',
+    description: 'Endpoint Security Event Filters List',
+  },
+  hostIsolationExceptions: {
+    id: 'endpoint_host_isolation_exceptions',
+    name: 'Endpoint Security Host isolation exceptions List',
+    description: 'Endpoint Security Host isolation exceptions List',
+  },
+  blocklists: {
+    id: 'endpoint_blocklists',
+    name: 'Endpoint Security Blocklists List',
+    description: 'Endpoint Security Blocklists List',
```

Comment:
```suggestion
    name: 'Endpoint Security Blocklist List',
    description: 'Endpoint Security Blocklist List',
```

Reply:
@ashokaditya - I remember having this conversation when we first introduced Blocklists. I think it needs to be plural, as each entry for Blocklist is a "list" of values.
Being that this is just a refactor, I'll leave it as is, unless you feel strongly about changing it.